person.rb

```diff
@@ -3,7 +3,6 @@
 require_relative 'trimmer_decorator'
 require_relative 'book'
 require_relative 'Rental'
```

Comment:
- Hey, great job so far, however, your app works perfectly fine but fails to start because of how specified the path for the rental. update it to `rental` and your app would work as expected.